knowledge-base/how-to-restrict-export-formats-in-preview.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to restrict export formats in Report Preview
+description: How to limit the rendering extensions in the Report Preview mode of the Report Server
+type: how-to
+page_title: How to restrict export formats
+slug: how-to-restrict-export-formats-in-preview
+position: 
+tags: 
+ticketid: 1408256
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Report Server</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+I want to hide some of the rendering extensions.
+
+## Solution
+By default, all extensions are visible in the viewer of the Report Server. To hide any of them it will be necessary to set accordingly the [extensions Element](https://docs.telerik.com/reporting/configuring-telerik-reporting-extensions) of the [Telerik Reporting Configuration](../implementer-guide/setup/configure-the-report-engine) section of the **TelerikReporting.config** file in the (_Telerik Report Server installation folder_)\\Telerik.ReportServer.Web (e.g. _C:\\Program Files (x86)\\Progress\\Telerik Report Server\\Telerik.ReportServer.Web_). After modifications the config file should look like :  
```

Comment:
I think it would be better if you specify the extensions (i.e rendering extensions in the export options of the viewer)